deepchem/feat/molecule_featurizers/one_hot_featurizer.py

```diff
@@ -16,14 +17,14 @@
 ]
 
 
-class OneHotFeaturizer(MolecularFeaturizer):
-  """Encodes SMILES as a one-hot array.
+class OneHotFeaturizer(Featurizer):
+  """Encodes SMILES or any arbitrary string as a one-hot array.
 
-  This featurizer encodes SMILES string as a one-hot array.
+  This featurizer encodes either a SMILES string or any arbitrary string as a one-hot array.
 
   Note
   ----
-  This class requires RDKit to be installed.
+  This class needs RDKit to be installed in order to work with RDKit molecules.
```

Comment:
This makes me even more confused!  I thought this featurizer operated on strings.  Where do RDKit molecules come into it?

Reply:
Fixed this comment to add more detail. Hope that helps.